src/utils/configuration.ts

```diff
@@ -7,9 +7,16 @@ export interface SnapshotConfiguration {
   'min-height'?: number,
 }
 
+export interface StaticSiteSnapshotConfiguration {
+  'base-url'?: string,
+  'snapshot-files'?: string,
```

Comment:
@djones maybe we name this `include-paths` and `exclude-paths`?

Reply:
These configuration names mirror the CLI flag names. I think it'd be good to keep those two consistent.